src/effects/RealtimeEffectList.cpp

```diff
@@ -8,6 +8,7 @@
 
 #include "RealtimeEffectList.h"
 #include "RealtimeEffectState.h"
+#include "RealtimeEffectStateUI.h"
```

Comment:
This `#include` is new since my last round of review, and unfortunately makes bad depenency cycles.

This is from a "before" picture made by scripts/graph.pl:
<img width="236" alt="Screen Shot 2022-07-28 at 4 36 18 PM" src="https://user-images.githubusercontent.com/11670369/181633036-fbdf035f-a809-43c2-8058-307edbfeb224.png">

Bad -- cycles crept in this release that we should fix later.  But let's not make them worse:


<img width="285" alt="Screen Shot 2022-07-28 at 4 33 14 PM" src="https://user-images.githubusercontent.com/11670369/181633774-1f0a9ce6-4ff0-4792-8274-286836ad37ea.png">

Remove the one `#include` and this bubble still grows by one, including the new file RealtimeEffectStateUI.  I'm not sure which is the edge to cut so that this doesn't happen.  But at least this does not tie AudioUI and the other Realtime* files.  My unmerged library extractions in progress would move those files down into lib-audio-io.

And how to do the rest of the code change to break the dependency is easy to see: the two functions of `RealtimeEffectList` that you modify already to `Publish` so you can make subscriptions elsewhere to handle this.

Reply:
Shoot, my bad! Will be fixed for sure!